teacher_class.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Teacher < Person
+  def
+    super(_name, _age)
+    @specialization
+  end
```

Comment:
- [ ] Kindly add the initialize method, the initialize method is a special method that gets called when an object is created from a class. It is commonly used to set up initial values for the instance variables of an object. 🙏